src/Render/utilities.jl

```diff
@@ -48,6 +48,72 @@ name(mod, q::QualifiedSymbol) = q.sym
 
 name(mod, aside::Aside) = symbol("")
 
+# Get the final name of an documented object such as ``Function`` or ``Method``.
+nameof(m::Module, obj) = name(m, obj)
+
+function nameof(m::Module, obj::Aside)
+    linenumber, path = Cache.getmeta(m, obj)[:textsource]
+    return symbol("aside_$(first(splitext(basename(path))))_L$(linenumber)")
+end
```

Comment:
Would be better to combine with the ``name`` method above on line 49. Just so long as the name we generate isn't in the exports list.